pkgs/tools/typesetting/asciidoc/default.nix

```diff
@@ -249,11 +254,13 @@ stdenv.mkDerivation rec {
         -e "s|^XMLLINT =.*|XMLLINT = '${libxml2.bin}/bin/xmllint'|" \
         -e "s|^EPUBCHECK =.*|EPUBCHECK = 'nixpkgs_is_missing_epubcheck'|" \
         -i a2x.py
-  '' + ''
-    for n in $(find "$out" . -name \*.py); do
-      sed -i -e "s,^#![[:space:]]*.*/bin/env python,#!${python2}/bin/python,g" "$n"
-      chmod +x "$n"
-    done
+  '' else ''
+    sed -e "s|^ENV =.*|ENV = dict(XML_CATALOG_FILES='${docbook_xml_dtd_45}/xml/dtd/docbook/catalog.xml ${docbook_xsl_ns}/xml/xsl/docbook/catalog.xml ${docbook_xsl}/xml/xsl/docbook/catalog.xml')|" \
```

Comment:
this first bit here shouldn't be needed. if the docbook dependencies are added to nativeBuildInputs you will have the setup hook `findXMLCatalogs` which will set this appropriately.

Reply:
@dasJ does this need to be addressed still? I don't have enough context to judge.

Reply:
**Probably** the same as my other comment